### PR TITLE
Login, Confirmation, Banner

### LINKED ISSUE #48: Login & confirmation screens - routing with no authentication
- [ ] Build only the screens with no authentication required
- [ ] Setup routing for login and confirmed

#### login screen
Screen Shot 2020-11-21 at 2 49 44 PM

#### confirmation screen
Screen Shot 2020-11-21 at 3 03 50 PM

### LINKED ISSUE #109: Banner component
Banner component used in authentication, application review, confirmation, etc (basically a styled div that is 20% transparent with controllable border radius)
- [ ] width should not be fixed - this should be controlled by the parent
- [ ] set margins and padding based on design specs
Screen Shot 2020-11-21 at 3 09 52 PM
Screen Shot 2020-11-21 at 3 10 14 PM
Screen Shot 2020-11-21 at 3 10 43 PM
Screen Shot 2020-11-21 at 3 10 58 PM

### PR DESCRIPTION
**Pull request by panjenny0**
Implemented these pages/components
- login with no authentication (closes #48)
- confirmation page (Note that sponsors will be added in another PR - thanks @michelleykim 🙏 )
- banner (closes #109)

Notes
- copied over routes for login/confirmation from #129 
Screen Shot 2020-11-27 at 12 31 38 AM
Screen Shot 2020-11-27 at 12 31 48 AM

#### Designs

Screen Shot 2020-11-27 at 12 36 13 AM
Screen Shot 2020-11-27 at 12 36 32 AM

### CODE REVIEW

File: src/containers/Landing/index.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  width: 100%;
+  top: 18em;
+  ${CenterHorizontally}
+  z-index: -1;
+`
+
+const StyledLogoLockup = styled.img`
+  position: absolute;
+  top: 7em;
+  margin: 0 50.5%;
```
Comment (panjenny0) [inline src/containers/Landing/index.js:31]: Note about this 50.5... the logo was not perfectly symmetrical by default so a 0.5 offset is needed.

#### before
Screen Shot 2020-11-27 at 12 30 15 AM

#### after
Screen Shot 2020-11-27 at 12 30 29 AM

File: src/components/Button.js [hunk 2]
```diff
@@ -112,6 +113,26 @@ const StyledButton = styled.a`
     :focus {
       box-shadow: 0 0 0 .2rem ${hexToRgba(p.theme.colors.primary, 0.5)};
     }`}
+    ${p =>
+    p.color !== 'primary' &&
+    p.color !== 'secondary' &&
+    p.color !== 'tertiary' && // some color other than the variants
```
Comment (ianmah) [inline src/components/Button.js:119]: Is this to change the default button color? Would this work as well?
```suggestion
    !p.color
```

File: src/components/Button.js [hunk 3]
```diff
@@ -112,6 +113,26 @@ const StyledButton = styled.a`
...
+    `color: ${p.labelColor ?? p.theme.colors.background};
+      background: ${p.color};
+      :hover {
+        ${
+          p.disabled
+            ? `
+        cursor: not-allowed;
+        `
+            : `
+        background: ${p.hover ?? p.theme.colors.tertiaryHover};
```
Comment (ianmah) [inline src/components/Button.js:129]: I think we've mostly been using `||`, could we use that here as well?

File: src/containers/Landing/index.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  transform: translateX(-50%);
+  z-index: 200;
+`
+
+const StyledBanner = styled(Banner)`
+  && {
```
Comment (ianmah) [inline src/containers/Landing/index.js:37]: What is this for?

**Review by ianmah — COMMENTED** [review]
Should we have a scrollbar on the login page if theres no content below?

File: src/components/Button.js (hunk 2 above)
Comment (panjenny0) [reply src/components/Button.js:119]: This is to change the button colour to a specified `color` (line 121) that is not one of the defined colours

File: src/components/Button.js (hunk 3 above)
Comment (panjenny0) [reply src/components/Button.js:129]: yupp

File: src/containers/Landing/index.js (hunk 4 above)
Comment (panjenny0) [reply src/containers/Landing/index.js:37]: I was having issues styling the `Banner` component so `&& {}` was a workaround I found https://github.com/styled-components/styled-components/issues/1816

**Comment by panjenny0** [comment]
> Should we have a scrollbar on the login page if theres no content below?

@ianmah ok fixed this holy it finally worked 💀 can you review again please & thank you!

**Review by ianmah — APPROVED** [review]

**Pull request merged**